packages/tupaia-web/src/theme/GlobalStyle.tsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import { Theme } from '@material-ui/core';
+import { createGlobalStyle } from 'styled-components';
+
+/**
+ * These are global styles used in the app. They are overrides that might not otherwise be possible to control at a component level, e.g. the datepicker dialog, as this actually appears outside of the component in the DOM
+ */
+export const GlobalStyle = createGlobalStyle<{
+  theme: Theme;
+}>` 
+  #date-picker-dialog {
```

Comment:
I would prefer to keep these styles in the component file of DateRangePicker. I assume that you added them here as you weren't able to style the modal in the DateRangePicker. It is actually possible using a a round about way which is to make a styled Paper component and pass that into the Mui Dialog component. Have a look at tupaia-web -> Modal.tsx for an example. We will also have to update the ui-components DatePicker Dialog to pass through the Paper component but I think that's still a good approach.